beets/library.py

```diff
@@ -418,6 +418,14 @@ class Item(LibModel):
         'lyricist':             types.STRING,
         'composer':             types.STRING,
         'composer_sort':        types.STRING,
+        'parent_work':          types.STRING,
+        'parent_work_disambig': types.STRING,
+        'work':                 types.STRING,
+        'work_disambig':        types.STRING,
+        'performer':            types.STRING,
+        'performer_sort':       types.STRING,
+        'parent_composer':      types.STRING,
+        'parent_composer_sort': types.STRING,
```

Comment:
If I just naively remove this part, I get: 
Traceback (most recent call last):
  File "/usr/bin/beet", line 11, in <module>
    load_entry_point('beets==1.4.4', 'console_scripts', 'beet')()
  File "/usr/lib/python3.6/site-packages/beets/ui/__init__.py", line 1228, in main
    _raw_main(args)
  File "/usr/lib/python3.6/site-packages/beets/ui/__init__.py", line 1215, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "/usr/lib/python3.6/site-packages/beets/ui/commands.py", line 933, in import_func
    import_files(lib, paths, query)
  File "/usr/lib/python3.6/site-packages/beets/ui/commands.py", line 910, in import_files
    session.run()
  File "/usr/lib/python3.6/site-packages/beets/importer.py", line 325, in run
    pl.run_parallel(QUEUE_SIZE)
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 445, in run_parallel
    six.reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/usr/lib/python3.6/site-packages/six.py", line 686, in reraise
    raise value
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 312, in run
    out = self.coro.send(msg)
  File "/usr/lib/python3.6/site-packages/beets/util/pipeline.py", line 194, in coro
    func(*(args + (task,)))
  File "/usr/lib/python3.6/site-packages/beets/importer.py", line 1398, in plugin_stage
    func(session, task)
  File "/usr/lib/python3.6/site-packages/beets/plugins.py", line 124, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/beetsplug/parentwork.py", line 59, in imported
    self.find_work(task.items)        
  File "/usr/lib/python3.6/site-packages/beetsplug/parentwork.py", line 135, in find_work
    item.store(fields=['parent_work','parent_work_disambig','work','work_disambig','performer','performer_sort','parent_composer','parent_composer_sort'])
  File "/usr/lib/python3.6/site-packages/beets/library.py", line 326, in store
    super(LibModel, self).store(fields)
  File "/usr/lib/python3.6/site-packages/beets/dbcore/db.py", line 381, in store
    tx.mutate(query, subvars)
  File "/usr/lib/python3.6/site-packages/beets/dbcore/db.py", line 693, in mutate
    cursor = self.db._connection().execute(statement, subvals)
sqlite3.OperationalError: no such column: parent_work